<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>borrowbook</class>
 <widget class="QDialog" name="borrowbook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="bookIdLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Book ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="memberIdLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Member ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="bookNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QLabel" name="authorLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QLabel" name="genreLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QLabel" name="coverImageLabel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>221</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="firstNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QLabel" name="lastNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="issuanceDateEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>110</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="returnDateEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>480</y>
     <width>110</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Issue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Is the book available?</string>
   </property>
  </widget>
  <widget class="QLabel" name="availabilityLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchBookButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Search Book</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchMemberButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Search Member</string>
   </property>
  </widget>
  <widget class="QPushButton" name="issueBookButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>560</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Issue Book</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="notesTextEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>370</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Notes</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="descriptionTextEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewContainer">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>370</y>
     <width>341</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>640</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>600</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Issue</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
